src/components/Alternative/Alternative.test.tsx

```
@@ -698,9 +698,9 @@ describe('AlternativesList', () => {
     });
   });
 
-  describe('Alternative.disabled in readonly mode (line 212)', () => {
+  describe('Alternative.disabled in readonly mode', () => {
     it('applies opacity-50 for disabled alternative in readonly mode', () => {
-      // Specific test for line 212: alternative.disabled ? 'opacity-50' : ''
+              // Specific test for disabled alternative styling
```

ESLint

[error] Delete `········`

(prettier/prettier)